app/lib/constants/site_config.rb

```diff
@@ -339,6 +339,10 @@ module SiteConfig
         description: "The \"API secret key\" portion of consumer keys in the Twitter developer portal.",
         placeholder: ""
       },
+      user_considered_new_days: {
+        description: "The number of days a user is considered new. Use 0 to disable these checks.",
```

Comment:
> Use 0 to disable these checks

I read this a few times and I have a feeling that this might not be clear to some less technical users (at least, I can imagine some users thinking, _"disable what check?"_). Maybe this could be:
```suggestion
        description: "The number of days a user is considered new. The default is 3 days, but you can disable this entirely by inputting 0.",
```

I don't _love_ my suggestion, so feel free to change it; just trying to think of ways that this user-facing text could potentially be misinterpreted or misunderstood by a Forem Creator 😅